scripts/std-lib-regression.sh

```diff
@@ -39,6 +39,19 @@ fi
 cargo new std_lib_test --lib
 cd std_lib_test
 
+# Add some content to the rust file including an std function that is non-generic.
+echo '
+#[kani::proof]
+fn check_format() {
+    assert!("2021".parse::<u32>().unwrap() == 2021);
+}
+' > src/lib.rs
+
+# Until we add support to this via our bundle, rebuild the kani library too.
+echo "
+kani = {path=\"${KANI_DIR}/library/kani\"}
```

Comment:
This might be a bit fragile since it depends on adding `kani` under the `[dependencies]` section. Perhaps in a future PR, we can rely on https://github.com/killercup/cargo-edit instead?

Reply:
I agree that this is rather fragile. TBH, I'm really hoping this script will be temporary. We need a better story for `std` distribution. Once we do have that, this script should go away and we will likely build (and distribute) the std.